opencog/query/CMakeLists.txt

```diff
@@ -1,3 +1,30 @@
+IF (HAVE_GEARMAN)
+IF (HAVE_GUILE)
+	ADD_LIBRARY(dist-gearman SHARED
+	    Distributed/DistSCM.cc
+	)
+	ADD_DEPENDENCIES(dist-gearman
+		opencog_atom_types
+	)
+	TARGET_LINK_LIBRARIES(dist-gearman
+	    ${GEARMAN_LIBRARY}
+	    atomspace
+	)
+	TARGET_LINK_LIBRARIES(dist-gearman
+		smob
+	)
+	INSTALL (TARGETS dist-gearman DESTINATION "lib${LIB_DIR_SUFFIX}/opencog")	
+	INSTALL (FILES
+		Distributed/DistSCM.h
+		DESTINATION "include/opencog/query"
+	)
+ELSE(HAVE_GUILE)
+	MESSAGE(FATAL_ERROR "Cannot find guile")
+ENDIF (HAVE_GUILE)
+
+ELSE(HAVE_GEARMAN)
+	MESSAGE(FATAL_ERROR "Cannot find libgearman library")
+ENDIF(HAVE_GEARMAN)
```

Comment:
Also, can you move this closer to the bottom of the file, so it gets built after everything else?